docs/src/lib/representations.md

```diff
@@ -201,6 +201,21 @@ Inherited from [`LazySet`](@ref):
 * [`radius`](@ref radius(::LazySet, ::Real))
 * [`diameter`](@ref diameter(::LazySet, ::Real))
 
+## HParallelotope
+
+```@docs
+HParallelotope
+directions(::HParallelotope{N}) where {N<:Real}
+offset(::HParallelotope{N}) where {N<:Real}
```

Comment:
Hm, I am not sure we need these getters because they seem to be type specific. Is there any chance that other set types would have a method with the same name? If not, we can just access the fields.
(Of course we can keep them now that you already added them. See this just as a general comment.)

Reply:
It's a valid point; these getter functions are type-specific. We may not want to `export` them. If we add another parallelotope type, the getter function may have to do some computation (compare to `base_vertex`, which is for `HParallelotope` some function, but for another set  type, say `GParallelotope` it could be a getter function).

Reply:
> If we add another parallelotope type, the getter function may have to do some computation

Great, then they are completely reasonable to have and I take back what I said.